src/common/metrics/case_density.tsx

```diff
@@ -136,19 +136,7 @@ function renderDisclaimer(): React.ReactElement {
       <ExternalLink href="https://ethics.harvard.edu/files/center-for-ethics/files/key_metrics_and_indicators_v4.pdf">
         “Key Metrics for Covid Suppression”
       </ExternalLink>{' '}
-      by Harvard Global Health Institute and others.
-      <br />
-      When estimating the number of people who will become infected in the
-      course of a year, we rely on the{' '}
```

Comment:
We should keep an eye on the inbox report for if we get questions about the estimated infections without this reference anymore.  I'd guess most people don't understand the difference between cases and infections.

Reply:
👍